hw/audio/ac97.c

```diff
@@ -291,6 +297,7 @@ static void reset_bm_regs (AC97LinkState *s, AC97BusMasterRegs *r)
     r->cr = r->cr & CR_DONT_CLEAR_MASK;
     r->bd_valid = 0;
 
+printf("bm_reset 0x%X\n", r - s->bm_regs);
```

Comment:
The commit with the debug changes would be removed before merge. They are only here for debugging purposes.

*(This particular message is also using an incorrect argument type, potentially causing stack corruption)*